### DISCUSSION TITLE
Sprint 2 meeting | 2022-10-04

### DISCUSSION THREAD

**Discussion by HuangJunye** [QPong/planning#13] [Meetings]
## Agenda

- [ ] Sprint review (10 min)
- [ ] Sprint retro (10 min)
- [ ] Sprint planning (10 min)
- [ ] #7 (20 min)
- [ ] Q&A (5 min)
- [x] Game dev resources and news (5 min)
  - I bought a PS5! (And PS5 just got jailbreak).

**Comment by HuangJunye** [comment]
## Review

[Last sprint planning](https://github.com/QPong/planning/discussions/11#discussioncomment-3690687)

### Dayeong

### Unathi

### Siddharth

### Junye 
- [x] #12
- [x] Define tasks with more details
- [ ] #7
- [x] Arrange 1:1 meetings with everyone
- [ ] #6 
- [x] Create Google Drive shared folder
- [ ] #30

**Reply by HuangJunye** [reply]
@tula3and @Unathi-Skosana @Siddharthgolecha Can you please post what you have done in the last sprint in the replies to this message before our sprint meeting? Thank you.

**Reply by tula3and** [reply]
I've just added comments of each issue below:
- #19 
- #20 
- #21 

Thank you!

**Comment by HuangJunye** [comment]
## Retro

### What went well


### What didn't go well


### What to improve?

**Reply by Siddharthgolecha** [reply]
**What went well?**
All the ideas were discussed thoroughly with an emphasis on implementation.
**What didn't go well?**
Nothing much
**What to improve?**
We can be a bit more strict with timings

**Reply by tula3and** [reply]
### What went well

- Define what to do

### What didn't go well

- Some obstructions to proceed
- Didn't write a blog post

### What to improve?

- Share questions in Slack

**Reply by Unathi-Skosana** [reply]
What went well:

Pico-8 is great for prototyping

What didn't go well?

Paying for pico-8

What to improve?

Get familair with Lua and hacking pico-8

**Reply by HuangJunye** [reply]
What to improve
- Establish sizing of tasks and plan better
- Find a way to fund PICO-8
- Think about a way to get us up to speed with PICO-8
- Strict with meeting time
- Share questions in Slack between meetings

**Comment by HuangJunye** [comment]
## Planning

### Dayeong

### Unathi

### Siddharth

### Junye
